nservicebus/upgrades/asb-6to7.md

```diff
@@ -59,4 +59,15 @@ snippet:topology-selection-upgrade-guide
 
 When the `EndpointOrientedTopology` is selected, it is also necessary to configure [publisher names](/nservicebus/azure-service-bus/publisher-names-configuration.md), in order to ensure that subscribers receive event messages:
 
-snippet:publisher_names_mapping_upgrade_guide
\ No newline at end of file
+snippet:publisher_names_mapping_upgrade_guide
+
+## Keep connection string safe
+
+Transport guarantees backward compatibility between older transport versions (below to version 7) and newer versions (version 7 and above).  
+In order to be able to enable `UseNamespaceNameInsteadOfConnectionString` follow the next steps:
+- upgrade all endpoints to version 7 or above.
+- switch on `UseNamespaceNameInsteadOfConnectionString` feature one endpoint at time. 
```

Comment:
do people need to deploy all endpoints with v7 prior to enabling UseNamespaceNameInsteadOfConnectionString? if so that is not clear in the above